### PR TITLE
Update testing instruction

### PR DESCRIPTION
**Pull request by tedhtchang**
Improve testing instruction for contributors to run the unit tests.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,8 @@ We use pre-commit to make sure the code is consistently formatted. To make sure
 
 ### Testing
 
+> Some tests, i.e. test_ray_details,test_cluster_status,and test_wait_ready require an Openshift cluster with [Codeflare Stack](https://github.com/opendatahub-io/distributed-workloads/blob/main/Quick-Start.md#quick-start-guide-for-distributed-workflows-with-the-codeflare-stack) installed.
```
Comment (anishasthana) [inline README.md:32]:
```suggestion
> Some tests, i.e. test_ray_details, test_cluster_status, and test_wait_ready require an OpenShift cluster with the [Codeflare Stack](https://github.com/opendatahub-io/distributed-workloads/blob/main/Quick-Start.md#quick-start-guide-for-distributed-workflows-with-the-codeflare-stack) installed.
```

**Review by anishasthana — DISMISSED** [review]

**Review by anishasthana — COMMENTED** [review]
/lgtm

File: README.md [hunk 2]
```diff
@@ -29,6 +29,8 @@ We use pre-commit to make sure the code is consistently formatted. To make sure
 
 ### Testing
 
+> Some tests, i.e. test_ray_details, test_cluster_status, and test_wait_ready require an OpenShift cluster with the [Codeflare Stack](https://github.com/opendatahub-io/distributed-workloads/blob/main/Quick-Start.md#quick-start-guide-for-distributed-workflows-with-the-codeflare-stack) installed.
```
Comment (MichaelClifford) [inline README.md:32]: I don't think this is true. Unless something has changed recently, all of the unit tests can be run locally **without** an OpenShift Cluster. 

We use mocker to fake any required OC responses.

**Review by MichaelClifford — CHANGES_REQUESTED** [review]

**Comment by tedhtchang** [comment]
Thanks @MichaelClifford. I verified again they don't need a real cluster.

**Review by MichaelClifford — APPROVED** [review]

**Comment by MichaelClifford** [comment]
LGTM @tedhtchang thanks for the changes :)

**Pull request merged**